### PR TITLE
Add Samsung J3 (SM-J320F), codename `j3xlte`

### PR DESCRIPTION
**Pull request by mardy**
Change-Id: Ie609c6f3ca7eabeea40e19e5c9d3e4e5b500944b

### CODE REVIEW

**Comment by ghost** [comment]
hey man, mind if I contribute in this port

**Comment by mardy** [comment]
> hey man, mind if I contribute in this port

You are more than welcome! I'm preparing a blog post where I explain in detail what I've done so far; when it's ready I'll post a link here. :-)

**Comment by mardy** [comment]
Here's a link with my notes on this port: http://www.mardy.it/blog/2019/07/notes-on-porting-the-samsung-j3-to-halium-%2B-ubports.html

File: manifests/samsung_j3xlte.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  <project path="vendor/samsung/common" name="mardy/android_vendor_samsung_common" remote="github" revision="cm-14.1" />
+  <project path="vendor/sprd" name="mardy/android_vendor_sprd" remote="github" revision="cm-14.1" />
+
+  <project path="device/common" name="lineageos/android_device_common" revision="refs/heads/staging/cm-14.0" remote="github" />
+  <project path="kernel/samsung/sharkls" name="mardy/android_kernel_samsung_sharkls" remote="github" revision="cm-14.1" />
+
+  <project path="bootable/recovery-twrp" name="omnirom/android_bootable_recovery" remote="github" revision="android-7.1" />
+
+  <project path="system/bt" name="android_system_bt" remote="los" revision="cm-14.1" />
```
Comment (lnjX) [inline manifests/samsung_j3xlte.xml:14]: Please use the style of the other files:
https://github.com/Halium/halium-devices/blob/halium-7.1/manifests/amazon_thor.xml

**Comment by Vince1171** [comment]
Hi, thank for your interest in the Halium project
is this PR still relevant/up-to-date?
if yes, could you change it as requested
if nothing is done in a week, I'll close this PR

**Comment by mardy** [comment]
> Hi, thank for your interest in the Halium project
> is this PR still relevant/up-to-date?
> if yes, could you change it as requested
> if nothing is done in a week, I'll close this PR

Oh, I forgot about this. I can fix the style issues, but I won't be working on this port for the foreseeable future. Does it still make sense to merge this?

**Comment by Vince1171** [comment]
> Oh, I forgot about this. I can fix the style issues, but I won't be working on this port for the foreseeable future. Does it still make sense to merge this?
don't worry about the style, I'll do a big commit this week to fix all indentation/style errors/mistakes on all manifests
It make sense so someone else could continue the port

**Comment by Vince1171** [comment]
if you could just remove the twrp project from the manifest, so I'll merge it

**Comment by mardy** [comment]
> if you could just remove the twrp project from the manifest, so I'll merge it

There you go :-)

**Comment by Vince1171** [comment]
wonderful

**Pull request merged**